jaeger_client/sampler.py

```diff
@@ -264,7 +264,7 @@ def __init__(self, strategies, max_operations):
         self.max_operations = max_operations
 
     def is_sampled(self, trace_id, operation=''):
-        sampler = self.samplers.get(operation, None)
+        sampler = self.samplers.get(operation)
         if sampler is None:
```

Comment:
a) can you add a test reproducing the exact exception the users got? I added it to the description.
b) should the above be `if sampler:` instead ? Otherwise I don't see what's really changing here.